.github/workflows/build.yml

```diff
@@ -98,3 +98,28 @@ jobs:
         run: cd .repo && npx projen package:js
       - name: Collect js Artifact
         run: mv .repo/dist dist
+  package-python:
+    needs: build
+    runs-on: ubuntu-latest
+    permissions: {}
+    if: "! needs.build.outputs.self_mutation_happened"
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+      - uses: actions/setup-python@v3
+        with:
+          python-version: 3.x
+      - name: Download build artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: build-artifact
+          path: dist
+      - name: Prepare Repository
+        run: mv dist .repo
+      - name: Install Dependencies
+        run: cd .repo && yarn install --check-files --frozen-lockfile
+      - name: Create python artifact
+        run: cd .repo && npx projen package:python
+      - name: Collect python Artifact
+        run: mv .repo/dist dist
```

Comment:
Is this going to overwrite or be overwritten by the `package-js` step? Maybe related.. I noticed the build artifact in the PR checks is only typescript

```
> tree ~/Downloads/build-artifact -L 1
/Users/hansonlu/Downloads/build-artifact
├── API.md
├── CODE_OF_CONDUCT.md
├── LICENSE
├── NOTICE
├── README.md
├── coverage
├── lib
├── package.json
├── src
├── test
├── test-reports
├── tsconfig.dev.json
├── tsconfig.json
└── yarn.lock
```

Reply:
I don't think so. It generates two separate sets of artifacts. One with the Python bindings and one with the Typescript bindings.